scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -213,7 +226,12 @@ function doImportProject() {
 
   local setup_templ="${setup_templ_dir}/${setup_templ_name}"
   local setup_ws="${setup_ws_dir}/${setup_ws_name}"
-  
+
+  if doIsWindows # cut drive letter from Windows path only (not neccesarry under macOS/Linux)
+  then
+    proj_dir=${proj_dir:2} # for example 'C:/Users/' becomes '/Users/'
+  fi
+
   echo "SETUP_TEMPL: ${setup_templ}" #DEBUG
   echo "PROJ_NAME:   ${proj_name}" #DEBUG
   echo "PROJ_DIR:    ${proj_dir}" #DEBUG
```

Comment:
where do these debug outputs come from? They are not present on master branch but for some reason not show as added within this PR. Github bug?
Should actually not make it into the release...

Reply:
I added them in an earlier commit ... 
Might still be useful for testing on macOS/Linux I thought (branch is still WIP).